hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/entities/AttributeValueConverter.java

```diff
@@ -0,0 +1,31 @@
+package org.hypertrace.trace.reader.entities;
+
+import io.reactivex.rxjava3.core.Single;
+import org.hypertrace.core.attribute.service.v1.LiteralValue;
+import org.hypertrace.entity.data.service.v1.AttributeValue;
+import org.hypertrace.entity.data.service.v1.Value;
+
+class AttributeValueConverter {
```

Comment:
It feels this is stateless and one might not need to mock it so how about making it final?

Reply:
Actually will go static for the same reasoning as the entity converter.